src/guessTheNumber.js

```diff
@@ -0,0 +1,33 @@
+let mysteryNumber;
+let docInput;
+
+function setNumber() {
+  docInput =  document.getElementById('answer').value
+  if(!mysteryNumber)
+    mysteryNumber = (Math.floor(Math.random() * 100) + 1).toString()
+}
+function combined() {
+  setNumber()
+  guessTheNumber(docInput, mysteryNumber)
+}
+
+function guessTheNumber(docInput, mysteryNumber) {
+  console.log('mystery number ' + mysteryNumber)
+  let input = docInput
```

Comment:
From Zeze: just wondering what the difference is between the variable "input" and the variable "docInput". Could the code have run just as well without declaring a second variable?